std-bits/table/src/main/java/org/enso/table/data/column/storage/numeric/AbstractLongStorage.java

```diff
@@ -67,7 +71,12 @@ public double doDouble(
               @Override
               public Long doLong(
                   long in, long arg, int ix, MapOperationProblemBuilder problemBuilder) {
-                return in + arg;
+                try {
+                  return Math.addExact(in, arg);
```

Comment:
Have you benchmarked this? Just curious about the overhead.

Reply:
Good idea. I added a benchmark and posted the results in a comment.

I think the results are satisfactory - there seems to be no overhead on the happy path and in case of overflow the slowdown I got was about 2.6x for 20% of rows overflowing (practical slowdown will depend on many factors - like % of rows overflowing or stack size).